website/content/docs/release-notes/consul-k8s/v1_1_x.mdx

```diff
@@ -0,0 +1,56 @@
+---
+layout: docs
+page_title: 1.1.x
+description: >-
+  Consul on Kubernetes release notes for version 1.1.x
+---
+
+# Consul on Kubernetes 1.1.0
+
+## Release Highlights
+
+- **Enhanced Envoy Access Logging:** Envoy access logs are now centrally managed via the `accessLogs` field within the ProxyDefaults CRD, to allow operators to easily turn on access logs for all proxies within the service mesh. 
+  
+- **Consul Envoy Extensions:** A new extension system has been added to Consul to modify Consul-generated Envoy resources outside of the Consul binary. This will allow extensions to add, delete, and modify Envoy listeners, routes, clusters, and endpoints, enabling support for additional Envoy features without changes to the Consul codebase. 
```

Comment:
```suggestion
- **Consul Envoy Extensions:** The new Envoy extension system enables you to modify Consul-generated Envoy resources outside of the Consul binary. This will allow extensions to add, delete, and modify Envoy listeners, routes, clusters, and endpoints, enabling support for additional Envoy features without changes to the Consul codebase. 
```